test/expected/mongo/dumper-output/deep-nested-js

```diff
@@ -0,0 +1,42 @@
+// This model was generated by Lumber. However, you remain in control of your models.
+// Learn how here: https://docs.forestadmin.com/documentation/v/v5/reference-guide/models/enrich-your-models
+const mongoose = require('mongoose');
+
+// This section contains the properties of your model, mapped to your collection's properties.
+// Learn more here: https://docs.forestadmin.com/documentation/v/v5/reference-guide/models/enrich-your-models#declaring-a-new-field-in-a-model
+const schema = mongoose.Schema({
+  'name': String,
+  'very': mongoose.Schema({
+    deep: mongoose.Schema({
+      model: mongoose.Schema({
```

Comment:
[MISC] I think we don't need `mongoose.Schema`. 
It's needed when the schema is used in many places.